source/includes/k8s/file-transfer-protocol-k8s.rst

```diff
@@ -0,0 +1,167 @@
+
+``SFTP`` access requires the same authentication as any other S3 client.
+MinIO supports the following authentication providers:
+
+- :ref:`MinIO IDP <minio-internal-idp>` users and their service accounts
+- :ref:`Active Directory/LDAP <minio-external-identity-management-ad-ldap>` users and their service accounts
+- :ref:`OpenID/OIDC <minio-external-identity-management-openid>` service accounts
+
+:ref:`STS <minio-security-token-service>` credentials **cannot** access buckets or objects over SFTP.
+To use STS credentials to authenticate, you must use an S3 API client or port.
+
+Authenticated users can access buckets and objects based on the :ref:`policies <minio-policy>` assigned to the user or parent user account.
+
+The SFTP protocol does not require any of the ``admin:*`` :ref:`permissions <minio-policy-mc-admin-actions>`.
+You may not perform other MinIO admin actions with SFTP.
+
+
+Prerequisites
+-------------
+
+- MinIO Operator v5.0.7 or later.
+- Enable an SFTP port (8022) for the server.
+- A port to use for the SFTP commands and a range of ports to allow the SFTP server to request to use for the data transfer.
+
+
+Procedure
+---------
+
+#. Enable SFTP for the desired Tenant:
+
+   .. tab-set::
+
+      .. tab-item:: Operator Console
+
+         - In the Operator Console, click on the tenant for which to enable SFTP.
+         - In the :guilabel:`Configuration` tab, toggle :guilabel:`SFTP` to :guilabel:`Enabled`.
+         - Click :guilabel:`Save`.
+         - Click :guilabel:`Restart` to restart MinIO and apply your changes.
+
+      .. tab-item:: Kubectl
+
+	 Use the following Kubectl command to edit the Tenant YAML configuration:
+
+	 .. code-block:: yaml
+
+            kubectl edit tenants/my-tenant -n my-tenant-ns
+
+	 Replace ``my-tenant`` and ``my-tenant-ns`` with the desired Tenant and namespace.
+
+         In the ``features:`` section, set the value of ``enableSFTP`` to ``true``:
+
+         .. code-block:: yaml
+
+            spec:
+              configuration:
+                name: my-tenant-env-configuration
+              credsSecret:
+                name: my-tenant-secret
+              exposeServices:
+                console: true
+                minio: true
+              features:
+                enableSFTP: true
+
+         Kubectl restarts MinIO to apply the change.
+
+         You may also set ``enableSFTP`` in your Helm chart or Kustomize configuration to enable SFTP for newly created Tenants.
+
+#. If needed, configure ingress or forwarding for the SFTP port according to your local policies.
+
+#. Use your preferred SFTP client to connect to the MinIO deployment.
+   You must connect as a user whose :ref:`policies <minio-policy>` allow access to the desired buckets and objects.
+
+   The specifics of connecting to the MinIO deployment depend on your SFTP client.
+   Refer to the documentation for your client.
+
+
+Examples
+--------
+
+The examples here use the ``sftp`` CLI client on a Linux system.
```

Comment:
```suggestion
The following examples use the ``sftp`` CLI client on a Linux system.
```

Reply:
Maybe even link to die.net? https://linux.die.net/man/1/sftp

Reply:
ooo good idea